src/draw/renesas/dave2d/lv_draw_dave2d_arc.c

```diff
@@ -53,8 +53,8 @@ void lv_draw_dave2d_arc(lv_draw_dave2d_unit_t * u, const lv_draw_arc_dsc_t * dsc
     // Generate render operations
     //
     d2_framebuffer(u->d2_handle,
```

Comment:
Should provide a helper similar to 
```
d2_framebuffer_from_draw_buf(u->d2_handle, draw_buf);
```

Noted the `buffer_area` could be optimized out.

Reply:
And below repeated changes could be optimized out.

Reply:
Same as in https://github.com/lvgl/lvgl/pull/5334#discussion_r1452464066

Reply:
I mean adding a helper function could simplify those long lines and won't break anything.

See 
https://github.com/lvgl/lvgl/blob/c7bece7d98a11a6fd017f5da0dea272c81b29e30/src/draw/vg_lite/lv_vg_lite_utils.c#L542-L563

Reply:
Added.